frontend_tests/node_tests/stream_data.js

```diff
@@ -491,3 +491,17 @@ zrequire('stream_data');
     page_params.notifications_stream = 'jazy';
     assert(!stream_data.notifications_in_home_view());
 }());
+
+(function test_remove_default_stream() {
+    var remove_me = {
+        stream_id: 674,
+        name: 'remove_me',
+        subscribed: false,
+        in_home_view: false,
+    };
+
+    stream_data.set_realm_default_streams([remove_me]);
+    stream_data.add_sub('remove_me', remove_me);
```

Comment:
You probably want to add the sub before setting default stream? It shouldn't matter, but it makes more sense.

Reply:
Yes.

Reply:
Yeah...sometimes it's interesting to know that things will work even with minimal setup, but I think we all agree here that explicitly setting up the sub probably makes sense.

Reply:
Done.